File: batch10amnav/TypesOfAdapter/app/src/main/java/com/example/typesofadapter/CardviewExample.java
package com.example.typesofadapter;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton;
import com.google.android.material.snackbar.Snackbar;
import com.google.android.material.textfield.TextInputEditText;

public class CardviewExample extends AppCompatActivity {
TextInputEditText emailedit,passwordedit;
ExtendedFloatingActionButton loginbtn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_cardview_example);
        emailedit = findViewById(R.id.emailedit);
        passwordedit = findViewById(R.id.passwordedit);
        loginbtn = findViewById(R.id.loginbtn);
        loginbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String emailtext = emailedit.getText().toString();


                Snackbar.make(loginbtn,"email => "+emailtext,Snackbar.LENGTH_LONG)
                        .setAction("Revers", new View.OnClickListener() {
                            @Override
                            public void onClick(View view) {
                                Toast.makeText(getApplicationContext(), "reverseclicked", Toast.LENGTH_SHORT).show();
                            }
                        }).show();
            }
        });
    }
}

File: kushalapp/BasickComponent/app/src/main/java/com/cbitss/basickcomponent/CardVIewExample.kt
package com.cbitss.basickcomponent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RadioButton
import android.widget.RadioGroup
import android.widget.Toast
import com.cbitss.basickcomponent.fragments.FirstFragment
import com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
import com.google.android.material.snackbar.Snackbar

class CardVIewExample : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_card_view_example)


        //Fragement open code
        val fragmentManager = supportFragmentManager
        val fragmenttrasction = fragmentManager.beginTransaction()
        fragmenttrasction.add(R.id.fragmentcontainer,FirstFragment())
        fragmenttrasction.commit()
    }
}